### DISCUSSION TITLE
Can't connect to shockbyte server

### DISCUSSION THREAD

**Discussion by IlluminateOW** [mircokroon/minecraft-world-downloader#223] [Q&A] [answered]
When I enter the ip for my shockbyte server, it gives me the error code: " Cannot connect to **.**.**.***:*****. The server may be down or on a different address. (java.net.UnknownHostException)."

However, I also tried it on cubecraft, and there it worked.

**Comment by mircokroon** [comment, marked as answer]
Are you sure you got the IP correct? Also make sure you give it the right port

**Reply by IlluminateOW** [reply]
Thanks for the reply! The server ip is correct, but how do I find the port?

**Reply by IlluminateOW** [reply]
OHHHHHHHHHHHHHHHHH. I just realized that I am incredibly stupid. I entered the server port in the server ip box, and didn't have the correct port in the port box (I used the default port). Your answer made me realize how it's actually supposed to be. I am not very experienced with these kinds of things, so I guess you could say that I typed in the wrong ip adress.

**Reply by WhynotFulp** [reply]
@IlluminateOW can you share screen shots of how you resolved this problem? I am experiencing something similar being unable to connect to local host while trying to download a realm.
